pyccel/codegen/printing/fcode.py

```diff
@@ -2221,7 +2241,7 @@ def _print_Assign(self, expr):
         lhs = expr.lhs
         rhs = expr.rhs
 
-        if isinstance(rhs, (FunctionCall, SetUnion, ListPop)):
+        if isinstance(rhs, (FunctionCall, SetUnion, ListPop, DictPopitem)):
             return self._print(rhs)
```

Comment:
So far I had not paid much attention to the fact that, under certain circumstances, we only print the RHS of an assignment. Is this because the LHS is effectively the `out` (or `inout`) parameter of the subroutine call on the RHS?